client/src/pages/Lobby/Lobby.tsx

```diff
@@ -93,6 +125,31 @@ const Lobby: FC<LobbyProps> = (props: LobbyProps) => {
             players={game.players}
             currentPlayerId={currentPlayerId}
           />
+          <div>
+            <h4>Cartes</h4>
+            {cards.map((card, index) => (
+              <span key={index} style={{ paddingRight: '1rem' }}>
+                {card}
+              </span>
+            ))}
+            {result && (
+              <>
+                <h4>{buildResultText(result, game.players)}</h4>
+                <h4>{`Il y avait ${result.count} oeuf(s).`}</h4>
```

Comment:
Don't use headings for styling purposes!

Reply:
I'll replace the h4 with a div + strong

Reply:
A `<p>` is better for texts! Or even nothing, just the `<strong>`!